templates/cli/lib/commands/pull.js.twig

```diff
@@ -253,19 +185,21 @@ const pullMessagingTopic = async () => {
 
 pull
     .command("project")
-    .description("Pulling your {{ spec.title|caseUcfirst }} project")
+    .description("Pulling your Appwrite project name")
     .action(actionRunner(pullProject));
 
 pull
     .command("function")
-    .description("Pulling your {{ spec.title|caseUcfirst }} cloud function")
-    .action(actionRunner(pullFunction))
+    .description(`Pulling your Appwrite functions`)
+    .option(`--yes`, `Flag to confirm all warnings`)
+    .option(`--all`, `Flag to pull all functions`)
+    .action(actionRunner(pullFunctions));
 
 pull
     .command("collection")
-    .description("Pulling your {{ spec.title|caseUcfirst }} collections")
+    .description("Pulling your Appwrite collections")
     .option(`--databaseId <databaseId>`, `Database ID`)
-    .option(`--all`, `Flag to pullialize all databases`)
+    .option(`--all`, `Flag to pull all databases`)
```

Comment:
Previous was `pullialize`. Let's do global search to make sure it isn't used anywhere else.